### PR TITLE
AliCloud Platform Instructions

### PR DESCRIPTION
**Pull request by dkumaras**
Signed-off-by: Deepa Kumaraswamy <dkumaras@progress.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by jonlives — APPROVED** [review]
Looks good to me!

File: docs-chef-io/content/inspec/platforms.md [hunk 1]
```diff
@@ -148,20 +132,77 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
...
+export ALICLOUD_ACCESS_KEY="anaccesskey"
+export ALICLOUD_SECRET_KEY="asecretkey"
+export ALICLOUD_REGION="eu-west-1"
+```
+
+### Train and InSpec Dependencies
+
+InSpec AliCloud depends on AliCloud SDK version 0.0.4, which is provided via Train AliCloud. InSpec does not ship with Train AliCloud and is explicitly listed in the Gemfile.
+
+### Executing Run Linting Test
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md]: We do not want to include any information about testing on the platform support page. That information is for developers.

File: docs-chef-io/content/inspec/platforms.md [hunk 2]
```diff
@@ -148,20 +132,77 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
...
+```bash
+$ bundle exec rake
+Running RuboCop...
+Inspecting 16 files
+................
+
+16 files inspected, no offenses detected
+```
+
+### Configuring Integration Tests
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md]: Please delete this section as it concerns testing.

File: docs-chef-io/content/inspec/platforms.md [hunk 3]
```diff
@@ -148,20 +132,77 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
...
+
+- Execute the `$ bundle exec rake test:setup_integration_tests` command.
+
+- You can run `terraform plan` command to view any changes required for your infrastructure. Rerun the same command to reinitialize your working directory if you have modified Terraform modules or backend configuration.
+
+- Execute the command `terraform plan -var-file=inspec-alicloud.tfvars.json -out inspec-alicloud.plan` to generate Terraform and InSpec variable files and the respective plan. The plan is saved in the `inspec-alicloud.plan` path
+
+- Execute the command `terraform apply "inspec-alicloud.plan"` to apply the plan.
+
+### Executing Integration Tests
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md]: Delete all testing sections.

File: docs-chef-io/content/inspec/platforms.md [hunk 4]
```diff
@@ -148,20 +132,77 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
...
+
+Execute following commands to run the integration tests:
+
+```bash
+$ bundle exec rake test:run_integration_tests
+
+bundle exec inspec exec test/integration/verify -t alicloud:// --input-file test/integration/build/alicloud-inspec-attributes.yaml --reporter cli json:inspec-output.json html:inspec-output.html --chef-license=accept-silent; rc=$?; if [ $rc -eq 0 ] || [ $rc -eq 101 ]; then exit 0; else exit 1; fi
+```
+
+### Destroying Created Resources Terraform
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md]: Delete this section.

File: docs-chef-io/content/inspec/platforms.md [hunk 5]
```diff
@@ -11,34 +11,25 @@ gh_repo = "inspec"
     weight = 30
 +++
 
-As of Chef InSpec 2.0, we have expanded our platform support beyond individual
-machines and now include support for select AWS and Azure resources.
+As of Chef InSpec 2.0, we have expanded our platform support beyond individual machines and now include support for select AWS, Azure, GCP, and AliCloud resources.
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md:14]: Generally, try not to include unrelated changes in your pull requests. This PR reformats the entire file.

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Please do not include testing instructions. Those instructions are intended for developers, not users.

Please remove the DS_Store files from the PR. We do not want them in the repo.

File: docs-chef-io/content/inspec/platforms.md [hunk 6]
```diff
@@ -148,20 +132,42 @@ If you have created a `~/.azure/credentials` file as above, you may also use the
...
+export ALICLOUD_ACCESS_KEY="anaccesskey"
+export ALICLOUD_SECRET_KEY="asecretkey"
+export ALICLOUD_REGION="eu-west-1"
+```
+
+### Train and InSpec Dependencies
+
+InSpec AliCloud depends on AliCloud SDK version 0.0.4, which is provided via Train AliCloud. InSpec does not ship with Train AliCloud and is explicitly listed in the Gemfile.
+
+### Destroying Created Resources Terraform
```
Comment (clintoncwolfe) [inline docs-chef-io/content/inspec/platforms.md]: Please remove this section. This is related to developer documentation for integration testing, not user documentation.

Comment (dkumaras) [reply docs-chef-io/content/inspec/platforms.md]: ok

Comment (dkumaras) [reply docs-chef-io/content/inspec/platforms.md]: done

**Review by clintoncwolfe — APPROVED** [review]
